src/container/facility/FacilityDetailImages.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Image } from 'grommet';
+import styled from 'styled-components';
+
+const DetailImage = styled(Image)`
+  display: flex;
+  flex: 50%;
+  height: calc(50% - 8px);
+  object-fit: cover;
+`;
+
+export const FacilityDetailImages = ({ images }) => {
+  return (
+    <Box
+      direction="row"
+      style={{
+        gap: '8px',
```

Comment:
Maybe it is better to adjust the `gap` ("small", etc) in the Grommet theme and then use
```
<Box gap="<type>"
```
This way we will be able to standardize gaps in the app.

Reply:
Later this will be arranged by this discussion: https://www.notion.so/windingtree/Code-improvement-ideas-7a0a4dd0723b4ba999d459968569195f?p=00c0daf191b04c3f9559bb2e89838cec&pm=s